src/styles/blocks/card.scss

```diff
@@ -0,0 +1,95 @@
+.product-card {
+  box-sizing: border-box;
+  padding-inline: 16px;
+  width: 200px;
+  background-color: $main-background-color;
```

Comment:
The variable `$main-background-color` is used here but it is not defined in this file. Make sure it is defined in a variables file or within this file to avoid compilation errors.